### PR TITLE
Mbashsiz os 13536 how to migrate to ceph

### PR DESCRIPTION
**Pull request by MortezaBashsiz**
I have added the migration to Ceph page

### CODE REVIEW

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+taxonomy:
+    category:
+        - docs
+---
+
+## How to migrate your storage from Quobyte or local storage to Ceph
+
+### Overview
+
+This document is a guide to help you to migrate your storage from Quobyte or local storage to Ceph. For this purpose, lets to define a scenario from real world.
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:13]: Typo: let's define

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+[~]># lsblk
+NAME  MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
+vda  254:0    0  50G  0 disk
+└─vda1 254:1  0  50G  0 part /
+vdb  254:16   0  10G  0 disk /var/lib/mysql
+vdc  254:32   0   2G  0 disk /home/www
+[~]>#
+```
+
+Quobyte also is able to give you storage as a block device same as your local devices and this is completely transparent from point of view of Operating System
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:36]: from the point of view of the Operating System

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+
+At this example, I have two volumes created with QB and attached to my Instance
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:38]: In this example

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+Jan 02 08:47:02 test-qb-to-ceph systemd[1]: Started MariaDB 10.3.36 database server.
+Jan 02 10:25:20 test-qb-to-ceph systemd[1]: Stopping MariaDB 10.3.36 database server...
+Jan 02 10:25:22 test-qb-to-ceph systemd[1]: mariadb.service: Succeeded.
+Jan 02 10:25:22 test-qb-to-ceph systemd[1]: Stopped MariaDB 10.3.36 database server.
+root@test-qb-to-ceph:~#
+```
+
+#### 2. Mount the new Ceph disks
+
+It is up to you where you would like to create temporary mount points. At this example, two directories created in ***/opt/db*** and ***/opt/web*** and disks are mounted
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:166]: In this example, two directories are created

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+tmpfs         3.9G  0  3.9G   0% /sys/fs/cgroup
+tmpfs         799M  0  799M   0% /run/user/1000
+/dev/vdd      15G  111M   14G   1% /opt/db
+/dev/vde      2.0G   71M  1.8G   4% /opt/web
+root@test-qb-to-ceph:~#
+```
+
+#### 5. Replace mount points
+
+All will be done in fstab, or you prefer to do it manually. And this is good idea to comment (not to delete) the old lines in fstab till you be sure about it
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:227]: until you are sure about it.

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+```shell
+root@test-qb-to-ceph:~# cat /etc/fstab
+# /etc/fstab: static file system information.
+UUID=919e4355-3134-483f-a80c-51f4516f2f3a    /    ext4    errors=remount-ro    0    1
+#UUID=a3ed0420-88ba-45a5-82e1-4bd120cdd3e6    /var/lib/mysql    ext4    errors=remount-ro    0    1
+UUID=7da89885-7d87-4a8b-a14d-4c6fa87cd2c6    /var/lib/mysql    ext4    errors=remount-ro    0    1
+#UUID=5ba23dfb-c1e2-404d-8eb6-10f39e859bac    /home/www    ext4    errors=remount-ro    0    1
+UUID=a7f03a6d-17a4-4b57-88d8-ea10b92b93fc    /home/www    ext4    errors=remount-ro    0    1
+root@test-qb-to-ceph:~#
+
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:238]: You'll need to mount the 2 mountpoints manually here, or reboot.

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+```
+
+#### 6. Check if it works
+
+Start services and check the functionality
+
+```shell
+root@test-qb-to-ceph:~# systemctl start nginx.service mariadb.service
```
Comment (cornuwel) [inline user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md:246]: Nitpick: you didn't start mariadb here

File: user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md (hunk 1 above)
Comment (MortezaBashsiz) [reply user/pages/03.Howtos/16.migrate-to-ceph/docs.en.md]: Fixed in last commit

**Pull request closed without merging**